.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "parser": "typescript",
+  "singleQuote": true,
+  "printWidth": 120,
```

Comment:
please remove this config file. see https://github.com/Thomaash/vis-configs

Reply:
We have to have a config which enables TS. I deleted the rest.